pkg/constants/constants.go

```diff
@@ -96,3 +96,8 @@ const (
 	// ProjectCloneDisable specifies that project cloning should be disabled.
 	ProjectCloneDisable = "disable"
 )
+
+const (
+	// This annotation is added to a PVC that is triggered by a scheduler to be dynamically provisioned. Its value is the name of the selected node.
```

Comment:
nit, according to [godoc comments](https://go.dev/doc/comment#const) format. This comment Should start with variable name
```suggestion
	// SelectedNodeAnnotation annotation is added to a PVC that is triggered by a scheduler to be dynamically provisioned. Its value is the name of the selected node.
```